go/processor/floria/precompiled.go

```diff
@@ -48,3 +54,215 @@ func getPrecompiledContract(address tosca.Address, revision tosca.Revision) (get
 	contract, ok := precompiles[common.Address(address)]
 	return contract, ok
 }
+
+////////////////////////////////////////////////////////////////////////////////
+// State precompiled contract
+
+// DriverAddress is the NodeDriver contract address
+var DriverAddress = tosca.Address(common.HexToAddress("0xd100a01e00000000000000000000000000000000"))
```

Comment:
```suggestion
var driverAddress = tosca.Address(common.HexToAddress("0xd100a01e00000000000000000000000000000000"))

func DriverAddress() tosca.Address { return driverAddress }
```
if I recall correctly, this would be a safer way of exposing a public const, that is not really a const. 
This could be applied to other public "consts"

Reply:
They are no consts because of the common.HexToAddress function which can not be called during compile time. Also they are public because they are used in the integration test package.

Reply:
it's fine that they are not consts, but maybe make them function ?